2020-05-08 20:36:53,831 ERROR 16144 [-/127.0.0.1/-/9ms GET /workorder/comfirm] nodejs.TypeError: Cannot read property 'data' of undefined
    at WorkorderService.comfirm (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:11:40)
    at WorkorderController.comfirm (C:\Users\FS\Desktop\egg\bs\bishe\app\controller\workorder.js:10:44)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:45:18)
    at C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:44:32)
    at C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:36:12
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at csrfAuth (C:\Users\FS\Desktop\egg\bs\bishe\app\middleware\csrfAuth.js:4:15)
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at fn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\loader\mixin\middleware.js:111:29)

pid: 16144
hostname: FS

2020-05-08 20:37:57,688 ERROR 6564 [-/127.0.0.1/-/10ms GET /workorder/comfirm] nodejs.TypeError: Cannot read property 'aggregate' of undefined
    at WorkorderService.comfirm (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:12:43)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 6564
hostname: FS

2020-05-12 21:23:43,444 ERROR 19944 [-/127.0.0.1/-/7ms POST /workorder/refuse] nodejs.TypeError: Cannot read property 'push' of undefined
    at WorkorderService.refuse (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:98:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 19944
hostname: FS

2020-05-12 21:24:36,044 ERROR 19944 [-/127.0.0.1/-/84ms POST /workorder/refuse] nodejs.MongoError: After applying the update to the document {_id: ObjectId('5eb1910c4130340504c44936') , ...}, the (immutable) field '_id' was found to have been altered to _id: ObjectId('5eb191594130340504c44937')
    at Function.create (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\core\error.js:44:12)
    at toError (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\utils.js:150:22)
    at coll.s.topology.update (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\operations\common_functions.js:376:39)
    at C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\core\connection\pool.js:420:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
driver: true
name: "MongoError"
index: 0
code: 66
errmsg: "After applying the update to the document {_id: ObjectId('5eb1910c4130340504c44936') , ...}, the (immutable) field '_id' was found to have been altered to _id: ObjectId('5eb191594130340504c44937')"
pid: 19944
hostname: FS

2020-05-12 21:25:12,250 ERROR 19944 [-/127.0.0.1/-/110ms POST /workorder/refuse] nodejs.MongoError: After applying the update to the document {_id: ObjectId('5eb1910c4130340504c44936') , ...}, the (immutable) field '_id' was found to have been altered to _id: ObjectId('5eb191594130340504c44937')
    at Function.create (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\core\error.js:44:12)
    at toError (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\utils.js:150:22)
    at coll.s.topology.update (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\operations\common_functions.js:376:39)
    at C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\core\connection\pool.js:420:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
driver: true
name: "MongoError"
index: 0
code: 66
errmsg: "After applying the update to the document {_id: ObjectId('5eb1910c4130340504c44936') , ...}, the (immutable) field '_id' was found to have been altered to _id: ObjectId('5eb191594130340504c44937')"
pid: 19944
hostname: FS

2020-05-12 21:26:27,039 ERROR 2180 [-/127.0.0.1/-/147ms POST /workorder/refuse] nodejs.MongoError: After applying the update to the document {_id: ObjectId('5eb1910c4130340504c44936') , ...}, the (immutable) field '_id' was found to have been altered to _id: ObjectId('5eb191594130340504c44937')
    at Function.create (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\core\error.js:44:12)
    at toError (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\utils.js:150:22)
    at coll.s.topology.update (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\operations\common_functions.js:376:39)
    at C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\core\connection\pool.js:420:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
driver: true
name: "MongoError"
index: 0
code: 66
errmsg: "After applying the update to the document {_id: ObjectId('5eb1910c4130340504c44936') , ...}, the (immutable) field '_id' was found to have been altered to _id: ObjectId('5eb191594130340504c44937')"
pid: 2180
hostname: FS

2020-05-12 21:27:51,693 ERROR 19468 [-/127.0.0.1/-/103ms POST /workorder/refuse] nodejs.TypeError: Cannot read property 'push' of undefined
    at WorkorderService.refuse (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:99:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 19468
hostname: FS

2020-05-12 21:28:34,164 ERROR 1820 [-/127.0.0.1/-/119ms POST /workorder/refuse] nodejs.TypeError: Cannot read property 'push' of undefined
    at WorkorderService.refuse (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:100:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 1820
hostname: FS

2020-05-13 10:54:59,990 ERROR 3004 [-/127.0.0.1/-/57ms GET /workorder/working] nodejs.MongoError: unknown argument to $lookup: locallField
    at Connection.<anonymous> (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\core\connection\pool.js:466:61)
    at Connection.emit (events.js:198:13)
    at processMessage (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\core\connection\connection.js:384:10)
    at Socket.<anonymous> (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\mongodb\lib\core\connection\connection.js:553:15)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at Socket.Readable.push (_stream_readable.js:224:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
ok: 0
errmsg: "unknown argument to $lookup: locallField"
code: 4571
name: "MongoError"
pid: 3004
hostname: FS

2020-05-13 11:17:36,314 ERROR 20776 [-/127.0.0.1/-/60ms GET /workorder/work] nodejs.ReferenceError: working is not defined
    at WorkorderService.work (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:156:25)
    at WorkorderController.work (C:\Users\FS\Desktop\egg\bs\bishe\app\controller\workorder.js:40:44)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:45:18)
    at middleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-jwt2\lib\index.js:145:11)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 20776
hostname: FS

2020-05-13 11:18:26,673 ERROR 20776 [-/127.0.0.1/-/49ms GET /workorder/work] nodejs.ReferenceError: working is not defined
    at WorkorderService.work (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:156:25)
    at WorkorderController.work (C:\Users\FS\Desktop\egg\bs\bishe\app\controller\workorder.js:40:44)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:45:18)
    at middleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-jwt2\lib\index.js:145:11)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 20776
hostname: FS

2020-05-14 16:49:58,432 ERROR 7300 [-/127.0.0.1/-/66ms POST /workorder/doComfirm] nodejs.SyntaxError: Identifier 'data' has already been declared
    at WorkorderController.doComfirm (C:\Users\FS\Desktop\egg\bs\bishe\app\controller\workorder.js:16:44)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:45:18)
    at middleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-jwt2\lib\index.js:145:11)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 7300
hostname: FS

2020-05-14 16:51:00,539 ERROR 7300 [-/127.0.0.1/-/2ms POST /workorder/doComfirm] nodejs.SyntaxError: Identifier 'data' has already been declared
    at WorkorderController.doComfirm (C:\Users\FS\Desktop\egg\bs\bishe\app\controller\workorder.js:16:44)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:45:18)
    at middleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-jwt2\lib\index.js:145:11)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 7300
hostname: FS

2020-05-14 16:59:47,940 ERROR 7300 [-/127.0.0.1/-/50ms GET /workorder/comfirm] nodejs.TypeError: Cannot read property 'itemPartition' of undefined
    at WorkorderService.comfirm (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:29:59)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 7300
hostname: FS

2020-05-14 17:00:57,667 ERROR 4884 [-/127.0.0.1/-/228ms GET /workorder/comfirm] nodejs.TypeError: Cannot read property 'itemPartition' of undefined
    at WorkorderService.comfirm (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:30:59)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 4884
hostname: FS

2020-05-14 17:04:41,940 ERROR 4884 [-/127.0.0.1/-/64ms GET /workorder/comfirm] nodejs.TypeError: Cannot read property 'itemPartition' of undefined
    at WorkorderService.comfirm (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:30:59)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 4884
hostname: FS

2020-05-14 17:06:47,508 ERROR 4884 [-/127.0.0.1/-/33ms POST /workorder/doComfirm] nodejs.SyntaxError: Identifier 'data' has already been declared
    at WorkorderController.doComfirm (C:\Users\FS\Desktop\egg\bs\bishe\app\controller\workorder.js:16:44)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:45:18)
    at middleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-jwt2\lib\index.js:145:11)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 4884
hostname: FS

2020-05-14 20:33:15,484 ERROR 13584 [-/127.0.0.1/-/32ms POST /workorder/workDetail?_id=5eace122ac01f5242ca03a08] nodejs.ReferenceError: tasks is not defined
    at WorkorderService.workDetail (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:213:28)
    at WorkorderController.workDetail (C:\Users\FS\Desktop\egg\bs\bishe\app\controller\workorder.js:46:44)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-compose\index.js:45:18)
    at middleware (C:\Users\FS\Desktop\egg\bs\bishe\node_modules\koa-jwt2\lib\index.js:145:11)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 13584
hostname: FS

2020-05-14 22:17:09,996 ERROR 21188 [-/127.0.0.1/-/488ms POST /servicerApply/servicerApply] nodejs.TypeError: Cannot read property 'filename' of undefined
    at ServicerApplyService.servicerApply (C:\Users\FS\Desktop\egg\bs\bishe\app\service\servicerApply.js:38:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 21188
hostname: FS

2020-05-14 22:17:39,909 ERROR 21188 [-/127.0.0.1/-/1583ms POST /servicerApply/servicerApply] nodejs.TypeError: Cannot read property 'filename' of undefined
    at ServicerApplyService.servicerApply (C:\Users\FS\Desktop\egg\bs\bishe\app\service\servicerApply.js:38:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 21188
hostname: FS

2020-05-14 23:50:43,942 ERROR 20288 [-/127.0.0.1/-/181ms POST /workorder/workDetail] nodejs.ReferenceError: ogData is not defined
    at WorkorderService.workDetail (C:\Users\FS\Desktop\egg\bs\bishe\app\service\workorder.js:227:40)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 20288
hostname: FS

